### ISSUE TITLE
[Bug] iPhone App blocked on load screen

### ISSUE THREAD

**Issue by giuseppelamatrice** [frappe/erpnext#10832]
App show ERPNext logo but **Desk** doesn't work (tested on iPhone 5 with iOS 10.3.3).

Solution: Delete and reinstall app (no good).

**Comment by strixaluco** [comment]
Duplicate of frappe/mobile/issues/8

**Comment by strixaluco** [comment]
By the way, would be great if you can participate in the bounty: https://www.bountysource.com/issues/48390608-bounty-for-fixing-bugs-and-improving-mobile-experience
Discussion: https://discuss.erpnext.com/t/bounty-fixing-bugs-and-improving-mobile-experience-of-erpnext-now-175/27282
